selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -178,8 +178,11 @@ AdvancedNetworking::AdvancedNetworking(QWidget* parent, WifiManager* wifi): QWid
   main_layout->addStretch(1);
 }
 
+void AdvancedNetworking::showEvent(QShowEvent *event) {
+  ipLabel->setText(wifi->getIp4Address());
```

Comment:
why remove this out of the refresh?

Reply:
This is a blocking call and I want to minimize the time it takes to refresh. Since ip is only displayed in the AdvanceNetworking, I feels it's reasonable to put it here.

Reply:
The downside is that if the ip changed, the user can only reopen an to see the new ip